vendor/bat-native-confirmations/src/bat/confirmations/internal/payout_tokens.cc

```diff
@@ -10,7 +10,7 @@
 #include "bat/confirmations/internal/unblinded_tokens.h"
 #include "bat/confirmations/internal/redeem_payment_tokens_request.h"
 
```

Comment:
no space here

Reply:
done

Reply:
did you push?

Reply:
we don't need another ci run for this fix so don't worry about that